build/templates/session.py.mako

```diff
@@ -100,6 +104,24 @@ class AttributeViSession(object):
 
     def __set__(self, obj, value):
         raise TypeError('Attributes of type ViSession are unsupported in Python')
+% for c in config['context_manager']:
+<%
+context_name = 'acquisition' if c['direction'] == 'input' else 'generation'
```

Comment:
Case doesn't matter here. I use title() case when I am using the name as a class name and lower() when I am using it as a member name.